cerberus/kubernetes/client.py

```diff
@@ -90,3 +92,27 @@ def monitor_component(iteration, component_namespace):
     logging.info("Iteration %s: %s: %s"
                  % (iteration, component_namespace, watch_component_status))
     return watch_component_status, failed_component_pods
+
+
+# Load env variables and initialize slack python client
+def initialize_slack_client():
+    global slack_client, slack_reporter_token, slack_channel_name
+    slack_reporter_token = os.environ["SLACK_API_TOKEN"]
```

Comment:
Better to add checks to see if the env variables used by the slack integration are defined and let user know if they are missing provided user enables this feature.

Reply:
Yeah, I will add it.